src/app/component/quiz/quiz.component.ts

```diff
@@ -67,6 +54,7 @@ export class Quizcomponent implements OnInit, OnDestroy {
   timer: any;
   selectedQuizType: any;
   radioValue: any;
```

Comment:
Could you please check and remove if not in use

Reply:
Yes i removed all unnecessary declarations